Source/JavaScriptCore/b3/air/AirAllocateRegistersAndStackAndGenerateCode.cpp

```diff
@@ -177,9 +177,12 @@ ALWAYS_INLINE void GenerateAndAllocateRegisters::flush(Tmp tmp, Reg reg)
     JIT_COMMENT(*m_jit, "Flush(", tmp, ", ", reg, ", offset=", offset, ")");
     if (tmp.isGP())
         m_jit->store64(reg.gpr(), callFrameAddr(*m_jit, offset));
-    else {
+    else if (B3::conservativeRegisterBytes(B3::FP) == sizeof(double) || !Options::useWebAssemblySIMD()) {
         ASSERT(m_map[tmp].spillSlot->byteSize() == bytesForWidth(Width64));
         m_jit->storeDouble(reg.fpr(), callFrameAddr(*m_jit, offset));
+    } else {
+        ASSERT(m_map[tmp].spillSlot->byteSize() == bytesForWidth(Width128));
+        m_jit->storeVector(reg.fpr(), callFrameAddr(*m_jit, offset));
```

Comment:
Ditto. We should avoid regression when they are used in pure JS context. We need FIXME and resolution before enabling SIMD.

Reply:
This allocator is only used for debugging and really huge wasm files, right?

Reply:
Does it regress the wasm code not using SIMD?

Reply:
I think all of the other cases should be handled automatically by the register allocator, but for this one, we either need a flag or we can take the regression. I don't actually think this register allocator is used in any of our benchmarks or in most real-world workloads, since you need to be in O0.

Anyway, for this patch it shouldn't matter yet